x-pack/plugins/apm/public/components/app/ServiceOverview/ServiceList/index.tsx

```diff
@@ -75,7 +75,22 @@ const SERVICE_COLUMNS = [
   }
 ];
 
-export function ServiceList({ items, noItemsMessage }) {
+// TODO: Can we just use camelCase in the server to avoid this mismatch, and avoid
+// having to magically transform keys?
```

Comment:
YES PLEASE! :) 
I did that exact change for the chart endpoints (https://github.com/elastic/kibana/pull/25514) and it's also an item on https://github.com/elastic/kibana/issues/19590

Reply:
Fixed this!